### DISCUSSION TITLE
Copy/Past

### DISCUSSION THREAD

**Discussion by goryachkinv** [radubirsan/FlashBrowser#10] [Q&A]
Guys!
Why I can't copy/past into browser? Definitely can't. 
My credential
macOS 12.2.1 Monterey
Apple M1

Who've any idea?

**Comment by Korne127** [comment]
Would have posted this as a new / own discussion, but then I saw this

System commands, e.g. command C to copy, command V to paste, command X to cut and command H to hide the windows all don't work on macOS. These commands don't need to implemented by a program but usually already work everywhere on default, so it's quite weird that none of these are working.
This is especially exhausting since it means you have to manually type long URLs (since you cannot copy / paste any of them)..
Can this be fixed?
(I would have posted this as an issue, but apparently they're not enabled. Why is that the case?)
